src/test/java/org/zalando/nakadi/controller/EventTypeControllerTestCase.java

```diff
@@ -103,7 +105,8 @@ public void init() throws Exception {
 
         final EventTypeService eventTypeService = new EventTypeService(eventTypeRepository, timelineService,
                 partitionResolver, enrichment, subscriptionRepository, schemaEvolutionService, partitionsCalculator,
-                featureToggleService, authorizationValidator, timelineSync, transactionTemplate, nakadiSettings);
+                featureToggleService, authorizationValidator, timelineSync, transactionTemplate, nakadiSettings,
+                nakadiKpiPublisher, "");
```

Comment:
maybe name it, because it is not clear what it is

Reply:
done https://github.com/zalando/nakadi/pull/799/commits/79da36f9ee405ae77b7e075811aeec25bcbab9d2